src/main/java/io/anserini/rerank/lib/AxiomReranker.java

```diff
@@ -109,14 +111,14 @@
   private final long seed;
   private final String originalIndexPath;
   private final String externalIndexPath;  // Axiomatic reranking can opt to use
-                                           // external sources for searching the expansion
```

Comment:
Can you undo these edits? I don't think they're supposed to be part of the PR?

Reply:
It seems like you ran a linter... which is fine. We can fix these minor issues... but the comments alignment should be reverted.